### PR TITLE
Email: Update billing frequency label for Titan

### PR DESCRIPTION
**Pull request by eltongo**
#### Changes proposed in this Pull Request

This updates the billing frequency label for Titan, from "Monthly billing" to "Annual or monthly billing".

#### Testing instructions

* Go to /email with the `email/titan-mvp` flag enabled
* Verify that the label has been updated

Screenshot 2020-10-15 at 11 13 25 AM

### CODE REVIEW

File: client/my-sites/email/email-providers-comparison/index.jsx [hunk 1]
```diff
@@ -10,25 +10,28 @@ import page from 'page';
...
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
 import formatCurrency from '@automattic/format-currency';
-import emailIllustration from 'assets/images/email-providers/email-illustration.svg';
-import titanLogo from 'assets/images/email-providers/titan.svg';
-import gSuiteLogo from 'assets/images/email-providers/gsuite.svg';
-import forwardingIcon from 'assets/images/email-providers/forwarding.svg';
+import emailIllustration from 'calypso/assets/images/email-providers/email-illustration.svg';
+import titanLogo from 'calypso/assets/images/email-providers/titan.svg';
+import gSuiteLogo from 'calypso/assets/images/email-providers/gsuite.svg';
+import forwardingIcon from 'calypso/assets/images/email-providers/forwarding.svg';
```
Comment (daledupreez) [inline client/my-sites/email/email-providers-comparison/index.jsx:34]: Yay linting... 🙄

File: client/my-sites/email/email-providers-comparison/index.jsx [hunk 2]
```diff
@@ -155,7 +158,7 @@ class EmailProvidersComparison extends React.Component {
 				) }
 				image={ { path: titanLogo } }
 				features={ [
-					translate( 'Monthly billing' ),
+					translate( 'Annual or monthly billing' ),
```
Comment (daledupreez) [inline client/my-sites/email/email-providers-comparison/index.jsx:161]: Is it worth flagging this for translation now? I am worried this won't get translated by Monday.

**Review by daledupreez — APPROVED** [review]
Other than handling fallout from the translation maybe not being ready in time for Monday, this LGTM.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4985104

Thank you @eltongo for including a screenshot in the description! This is really helpful for our translators.

**Review by daledupreez — APPROVED** [review]
LGTM! I tested this using German, UK English, and English, and everything looked good.

File: client/my-sites/email/email-providers-comparison/index.jsx [hunk 3]
```diff
@@ -3,32 +3,40 @@
  */
 import React from 'react';
 import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
+import i18n, { localize } from 'i18n-calypso';
```
[the remark this answers is not in the document]
Comment (eltongo) [reply client/my-sites/email/email-providers-comparison/index.jsx:6]: Yeah... FWIW the exact same import line is used throughout our codebase, that's why I went with it.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4985104).

**Pull request merged**